.coveragerc

```diff
@@ -8,7 +8,6 @@ show_missing = false
 precision = 0
 exclude_lines = true
 omit =
-    */src/buildercore/threadbare/*
```

Comment:
investigate state of coverage and if the below fixes anything

[run]
concurrency = gevent

Reply:
it's already in there ...